config/default_bpnode_fcncodes.h

```diff
@@ -0,0 +1,41 @@
+/************************************************************************
+ * NASA Docket No. GSC-18,719-1, and identified as “core Flight System: Bootes”
+ *
+ * Copyright (c) 2020 United States Government as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ * All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain
+ * a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ************************************************************************/
+
+/**
+ * @file
+ *   Specification for the BPNode command function codes
+ *
+ * @note
+ *   This file should be strictly limited to the command/function code (CC)
+ *   macro definitions.  Other definitions such as enums, typedefs, or other
+ *   macros should be placed in the msgdefs.h or msg.h files.
+ */
+#ifndef BPNODE_FCNCODES_H
+#define BPNODE_FCNCODES_H
+
+/************************************************************************
+ * Macro Definitions
+ ************************************************************************/
+
+/*
+** BPNode command codes
+*/
+#define BPNODE_NOOP_CC           0
+#define BPNODE_RESET_COUNTERS_CC 1
```

Comment:
Could I ask you to spell out all of the other function codes (from the M&C ICD) here?

Reply:
Also, I seem to remember that 582's rules for static analysis (with code sonar) will flag `#define`s that don't:

- wrap the value in parenthesis, or
- specify the type with (`u` or `U`) with unsigned constants

Could you update all of the `#defines` (here and elsewhere in the headers)?

Reply:
Will do

Reply:
Done